multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/auditlogging/CsrfTokenApiServiceAuditLog.java

```diff
@@ -0,0 +1,21 @@
+package org.cloudfoundry.multiapps.controller.core.auditlogging;
+
+import org.cloudfoundry.multiapps.controller.core.Messages;
+import org.cloudfoundry.multiapps.controller.core.auditlogging.model.AuditLogConfiguration;
+
+public class CsrfTokenApiServiceAuditLog {
+
+    private final AuditLoggingFacade auditLoggingFacade;
+
+    public CsrfTokenApiServiceAuditLog(AuditLoggingFacade auditLoggingFacade) {
+        this.auditLoggingFacade = auditLoggingFacade;
+    }
+
+    public void logGetInfo(String username) {
```

Comment:
String username a parameter almost everywhere. Why not just getting it in the methods in the ServiceAuditLog? This way you will have one parameter less